doc/conf.py

```diff
@@ -45,12 +45,21 @@
     "sphinx_reredirects",
 ]
 
+
+# Single file redirect are handled in this file and can be done by a pylint
+# contributor. See:
 # https://documatt.gitlab.io/sphinx-reredirects/usage.html
+# Directory redirect are handled in ReadTheDoc admin interface and can only be done
+# by pylint maintainer at the following URL:
+# https://readthedocs.org/dashboard/pylint/redirects/
 redirects: dict[str, str] = {
     # "<source>": "<target>"
     "intro": "index.html",
     "support": "contact.html",
     "user_guide/ide-integration": "installation.html",
+    "messages/messages_introduction": "user_guide/messages/index.html",
+    "messages/messages_list": "user_guide/messages/messages_overview.html",
+    "user_guide/message-control": "user_guide/messages/message-control.html",
```

Comment:
This seems like a good time to standardise either `word-word` or `word_word`.

Reply:
Let's use word_word, but let's do the normalization after we move the other one in the next MR.

Reply:
I'd prefer doing it now. You never know when somebody might copy a link to our documentation (perhaps this afternoon) and we would need to add another redirect for `message-control` > `message_control`.

Reply:
Also it means the temporary link to ``transform_feature`` must be maintained for all eternity after being live during the time to review the next MR.